mlflow/pyspark/ml/__init__.py

```diff
@@ -0,0 +1,134 @@
+import json
+
+import mlflow
+from mlflow.utils import chunk_dict, truncate_dict
+from mlflow.utils.annotations import experimental
+from mlflow.utils.autologging_utils import (
+    autologging_integration,
+    safe_patch,
+    try_mlflow_log,
+)
+
+FLAVOR_NAME = "pyspark_ml"
```

Comment:
Discussion:

we decided to put the pyspark ml autologging under namespace: `mlflow.pyspark.ml`
then:
* should we also move the `log_model` method into `mlflow.pyspark.ml` (deprecate old one) ?
* should we also change the other flavor name from "spark" to "pyspark_sql" ?